docs/gallery/plot_term_set.py

```diff
@@ -0,0 +1,146 @@
+"""
+TermSet
+=======
+
+This is a user guide for interacting with the
+:py:class:`~hdmf.TermSet` class. The :py:class:`~hdmf.TermSet` type
```

Comment:
See other comment that :py:class:`~hdmf.TermSet` doesn't link correctly.

Also, if we move these into an external_resources.py module, all of these links will have to be updated...